README.adoc

```diff
@@ -17,6 +17,8 @@
 :page-related-guides: ['microprofile-config', 'microprofile-metrics', 'cdi-intro', 'rest-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
+:page-seo-title: Checking the health of microservices using MicroProfile Health Check
```

Comment:
Change
"Checking the health of microservices using MicroProfile Health Check"
To
"Checking the health of microservices by using MicroProfile Health Check"